stdlib/asm/sys/mod.masm

```diff
@@ -33,3 +33,16 @@ end
 export.drop_stack_top
     dropw dropw dropw dropw
 end
+
+#! Logs a precompile commitment and removes the helper words produced by `log_precompile`.
+#!
+#! Input: `[COMM, TAG, ...]`
+#! Output: Stack without the three helper words `[R1, R0, CAP_NEXT]`.
+#! Cycles: 1 (plus cost of the underlying `log_precompile` instruction).
+export.log_precompile_request
```

Comment:
This is the procedure that is thoroughly tested to ensure that the log-precompile op-code works. It allows us to avoid the complicated end-to-end tests for the precompiles and only focus on the implementations of the wrappers, the handlers and verifiers.